composer.json

```diff
@@ -20,8 +20,8 @@
         "symfony/monolog-bundle": "~2.5",
         "symfony/monolog-bridge": "~2.4",
         "phpunit/phpunit": "~4.1",
-        "squizlabs/php_codesniffer": "~1.5",
-        "ongr/ongr-strict-standard": "~1.0.0"
+        "squizlabs/php_codesniffer": "~2.0",
```

Comment:
I'm not so sure if that's correct. The strict standard has a dependency in `require` section for code sniffer. So it definitely will be installed. 

https://github.com/ongr-io/ongr-strict-standard/blob/a0593e5f569664d9af0187bca0d2da7024ccf931/composer.json#L21